### PR TITLE
Increase resources for eventing-ceph 1.3 dot release

### PR DESCRIPTION
**Pull request by carlisia**
Looking at the prow failure for eventing-ceph: tests passed. Failing on the build step for  knative.dev/eventing-ceph/cmd/webhook for linux/amd64, seems some client-go informers might be requiring more mem than available for go build to run. 

Ref: https://prow.knative.dev/view/gs/knative-prow/logs/ci-knative-sandbox-eventing-ceph-1.3-dot-release/1502164255141531648

### CODE REVIEW

File: prow/jobs/config.yaml [hunk 1]
```diff
@@ -18707,6 +18707,7 @@ periodics:
       - "--auto-release"
       - "--release-gcs"
       - "knative-releases/eventing-kafka"
+      - "knative-releases/eventing-ceph"
```
Comment (dprotaso) [inline prow/jobs/config.yaml:18710]: This shouldn't be here - it's part of the `ci-knative-sandbox-eventing-kafka-auto-release` job

**Comment by dprotaso** [comment]
don't commit the `third_party` files you might have an older versions of `go-licenses` on your machine?

**Comment by carlisia** [comment]
what! where did that come from! fixing..

**Pull request merged**